packages/cache/__tests__/tar.test.ts

```diff
@@ -87,14 +94,16 @@ test('gzip extract tar', async () => {
       '-P',
       '-C',
       IS_WINDOWS ? workspace?.replace(/\\/g, '/') : workspace
-    ].concat(IS_MAC ? ['--delay-directory-restore'] : []),
+    ]
```

Comment:
Same with this, if this can be extracted out in a single utility/setup method.

Reply:
I think this can be left like this as combination might need changing with different conditions